workshop_app/tests/test_views.py

```diff
@@ -244,3 +260,13 @@ def setUp(self):
 						)
 	
 	
+class TestStaticPages(TestCase):
+
+	def test_testimonials(self): self.client.get('/testimonials/')
```

Comment:
those functions are just to test static pages , plus they follow pep8 standard so I don't think readability is an issue.